rules/docker-fpm-frr.mk

```diff
@@ -6,10 +6,10 @@ DOCKER_FPM_FRR_DBG = $(DOCKER_FPM_FRR_STEM)-$(DBG_IMAGE_MARK).gz
 
 $(DOCKER_FPM_FRR)_PATH = $(DOCKERS_PATH)/$(DOCKER_FPM_FRR_STEM)
 
-$(DOCKER_FPM_FRR)_DEPENDS += $(FRR) $(FRR_SNMP) $(SWSS) $(LIBYANG)
+$(DOCKER_FPM_FRR)_DEPENDS += $(FRR) $(FRR_SNMP) $(FRR_PYTHONTOOLS) $(SWSS) $(LIBYANG)
```

Comment:
Why do you need pythontools here? Do you have any dependancy on frr-reload.py?
It depends on python3 which will make the container bigger.

Reply:
Yes - `service frr reload` which can deal with incremental configuration, will call frr-reload.py

Reply:
In this case I think we should modify debian/control to make dependency of frr_pythontools on python2.

Reply:
This will cause a divergence from frr and I'm not sure how it can be extensively tested. In addition, main frr package also depends on python3 so it gets pulled in anyway.

Reply:
I see only build dependency
https://github.com/FRRouting/frr/blob/master/debian/control
But main package doesn't need python3

Can we fix control file and run pythontools with python2 ?